types/node/vm.d.ts

```diff
@@ -19,16 +20,26 @@ declare module "vm" {
          */
         columnOffset?: number;
     }
-    interface ScriptOptions extends BaseOptions {
-        displayErrors?: boolean;
-        timeout?: number;
-        cachedData?: Buffer;
+
+    interface ScriptOptionsObject extends BaseOptions {
+        cachedData?: NodeJS.TypedArray | DataView;
+        /** @deprecated This option is deprecated in favor of script.createCachedData() */
         produceCachedData?: boolean;
+        /** This option is part of the experimental API for the --experimental-modules flag, and should not be considered stable. */
+        importModuleDynamically?(specifier: string, module: SourceTextModule): Object | SourceTextModule;
```

Comment:
I don't think `Object` is what you want here. I assume `object` isn't available in the minimum TypeScript version for Node?

Reply:
I think that at the moment we can define this as ```any```, since the return type is quite specific